src/help.cpp

```diff
@@ -1033,6 +1034,38 @@ A: Ask the helpful people on the forum at smf.cataclysmdda.com or at the irc cha
     return text;
 }
 
+std::vector<std::string> text_bases()
```

Comment:
This function is never called.

Reply:
That's fine (see coothulu's help.cpp review)

Reply:
It means you should scrap the function, though.
You can recover it from git history later, don't keep it commented out.